src/main/resources/db/changelog/changes/v1.1.0/update-drv-purchase-order-line-details-definition.xml

```diff
@@ -59,6 +59,30 @@
               "valueGetter": "purchase_order.jsonb ->> 'poNumber'",
               "visibleByDefault": false
             },
+            {
+              "name": "po_workflow_status",
+              "dataType": {
+                "dataType": "stringType"
+              },
+              "valueGetter": "purchase_order.jsonb ->> 'workflowStatus'",
+              "filterValueGetter": "\"left\"(lower(${tenant_id}_mod_orders_storage.f_unaccent(purchase_order.jsonb ->> 'workflowStatus'::text)), 600)",
```

Comment:
Gotta love indexes that really don't make any sense 😆 There are only 3 possible values, all of which are short and unaccented 😛

Reply:
I thought the same thing lol